pkg/app/pipedv1/notifier/slack.go

```diff
@@ -205,20 +205,21 @@ func (s *slack) buildSlackMessage(event model.NotificationEvent, webURL string)
 		fields            []slackField
 	)
 
-	generateDeploymentEventData := func(d *model.Deployment, accounts string) {
+	generateDeploymentEventData := func(d *model.Deployment, accounts string, groups string) {
 		link = fmt.Sprintf("%s/deployments/%s?project=%s", webURL, d.Id, d.ProjectId)
 		fields = []slackField{
 			{"Project", truncateText(d.ProjectId, 8), true},
 			{"Application", makeSlackLink(d.ApplicationName, fmt.Sprintf("%s/applications/%s?project=%s", webURL, d.ApplicationId, d.ProjectId)), true},
 			{"Kind", strings.ToLower(d.Kind.String()), true},
 			{"Deployment", makeSlackLink(truncateText(d.Id, 8), link), true},
 			{"Triggered By", d.TriggeredBy(), true},
-			{"Mention To", accounts, true},
+			{"Mention To Users", accounts, true},
+			{"Mention To Groups", groups, true},
 			{"Started At", makeSlackDate(d.CreatedAt), true},
 		}
 	}
 
-	generateDeploymentEventDataForTriggerFailed := func(app *model.Application, hash, msg, accounts string) {
+	generateDeploymentEventDataForTriggerFailed := func(app *model.Application, hash, msg, accounts string, groups string) {
```

Comment:
[IMO]
How about calling getAccountsAsString(), getGroupsAsString() here to simplify the switch part?

```suggestion
	generateDeploymentEventDataForTriggerFailed := func(app *model.Application, hash, msg string, accounts, groups []string) {
	accountsStr := getAccountsAsString(accounts)
	groupsStr := getGroupsAsString(groups)
```

Reply:
@t-kikuc thank you, yes, calling that way should be visible!

Reply:
same question above?